src/client/components/segment-bubble/segment-bubble.tsx

```diff
@@ -16,64 +16,39 @@
  */
 
 import * as React from "react";
-import { Clicker, Dimension } from "../../../common/models/index";
-import { Fn } from "../../../common/utils/general/general";
 import { clamp } from "../../utils/dom/dom";
 import { BodyPortal } from "../body-portal/body-portal";
-import { SegmentActionButtons } from "../segment-action-buttons/segment-action-buttons";
 import { Shpitz } from "../shpitz/shpitz";
 import "./segment-bubble.scss";
 
 const OFFSET_V = -10;
 const PER_LETTER_PIXELS = 8;
+const MIN_TITLE_WIDTH = 80;
+const MAX_TITLE_WIDTH = 300;
 
 export interface SegmentBubbleProps {
   left: number;
   top: number;
-  dimension?: Dimension;
-  segmentLabel?: string;
-  measureLabel?: string;
-  clicker?: Clicker;
-  onClose?: Fn;
-  openRawDataModal?: Fn;
+  title: string;
+  content?: string | JSX.Element;
```

Comment:
Could we use only more specific type? This union looks weird, what is a common denominator of string and JSX.Element?

Reply:
`PropTypes` have JSXNode, I think we could define it as this union somewhere and replace usages.